buildenv

```diff
@@ -1,5 +1,8 @@
-export ZOPEN_DEV_URL="https://github.com/ZOSOpenTools/meta.git"
+# bump: meta-version /META_VERSION="(.*)"/ https://github.com/ZOSOpenTools/meta.git|semver:^0
+META_VERSION="0.8.0"
+
 export ZOPEN_BUILD_LINE="DEV"
+export ZOPEN_DEV_URL="https://github.com/ZOSOpenTools/meta.git"
```

Comment:
We may want to add the notion of a ZOPEN_STABLE release which is based on a specific release tag. @MikeFultonDev what do you think? is it still too early for that? Right now we're building purely off of the main line.

Reply:
It’s a good idea and yes - perhaps a bit early? But soon - this year